VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -39,31 +36,36 @@
     static public String CourseData = TAG + ".course_data";
 
     private EnrolledCoursesResponse courseData;
+    private GetCourseDiscussionInfoTask getCourseDiscussionInfoTask;
 
     @Inject
     IEdxEnvironment environment;
-
+    @InjectView(R.id.course_detail_name)
+    private TextView courseTextName;
+    @InjectView(R.id.course_detail_extras)
+    private TextView courseTextDetails;
+    @InjectView(R.id.header_image_view)
+    private ImageView headerImageView;
+    @InjectView(R.id.dashboard_detail)
+    private LinearLayout parent;
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_course_dashboard, container,
-                false);
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_course_dashboard, container, false);
+    }
 
-        courseTextName = (TextView) view.findViewById(R.id.course_detail_name);
-        courseTextDetails = (TextView) view.findViewById(R.id.course_detail_extras);
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
 
         //Implementation Note: - we can create a list view and populate the list.
         //but as number of rows are fixed and each row is different. the only common
         //thing is UI layout. so we reuse the same UI layout programmatically here.
-        LinearLayout parent = (LinearLayout)  view.findViewById(R.id.dashboard_detail);
-
-
+        LayoutInflater inflater = getLayoutInflater(savedInstanceState);
```

Comment:
The `getLayoutInflater()` method is not really part of the public API of `Fragment`, even though it's exposed in the support library. Therefore, it would be best to get the `LayoutInflater` using the `LayoutInflater.from()` method.